runner/runner/handlers.py

```diff
@@ -87,5 +82,5 @@ def _parse_container_logs(logs):
     },
     autoretry_for=(Exception,),
 )
-def publish_result(self, result, topic):
-    send_message(topic, "TASK_RESULT", result)
+def publish_result(self, result):
+    send_message("tasking.results", "TASK_RESULT", result)
```

Comment:
Is this hard code going to be permanent? or do we need a TODO here for later?

Reply:
No, this shouldn't be hardcoded.  This will probably be part of the info that the runner gets back during the registration process.  I'll add a TODO along those lines.

Reply:
TODO Added.